toolkits/springer/packages/springer-listing/scss/50-components/_listing.scss

```diff
@@ -95,3 +94,12 @@
 	margin-right: $listing--view-options-icon-margin-right;
 	fill: currentColor;
 }
+
+@supports (-webkit-line-clamp: 3) {
+	.c-listing__intro {
+		overflow: hidden;
+		display: -webkit-box;
+		-webkit-line-clamp: var(--listing-lines, 2);
```

Comment:
would we also need `line-clamp` here? For when it gets unprefixed.

(yay! Safari have just (finally) unprefixed `appearance`)